extensions/sql-database-projects/src/dialogs/deployDatabaseDialog.ts

```diff
@@ -92,10 +93,11 @@ export class DeployDatabaseDialog {
 					{
 						title: constants.targetDatabaseSettings,
 						components: [
+							/* TODO : enable using this when data source creation is enabled
```

Comment:
With this removal, does `connectionIsDataSource` need to be explicitly set to false (rather than undefined)?

Reply:
The two places its checked for 1) enable deploy/script button 2) chose connection  to create - both checking undefined  so it works. Let me know if you still feel I should change it (closing this PR but can do in the other open PR.)